src/test/java/org/folio/rest/impl/OaiPmhSetImplTest.java

```diff
@@ -71,58 +72,47 @@
     .withSetSpec("update SetSpec");
 
   private final Header tenantHeader = new Header("X-Okapi-Tenant", OAI_TEST_TENANT);
-  private final Header okapiUrlHeader = new Header("X-Okapi-Url", "http://localhost:" + okapiPort);
+  private final Header okapiUrlHeader = new Header("X-Okapi-Url", "http://localhost:" + mockPort);
```

Comment:
can probably make these static too

Reply:
or, since you're using `@TestInstance(PER_CLASS)` i *think* you could also just remove static from all these members.

Reply:
Thank you, corrected